blocks/core/ff_module/ff_module-button/_ff_module-button-renderer.js

```diff
@@ -6,7 +6,7 @@ var Button = require('./ff_module-button');
 var buttonProps = [{
     text: 'Button default',
     id: 'btn-01'
-}{
+},{
```

Comment:
This was causing a build error. Assume its fixed elsewhere, but just incase, it's here too.